Assets/KoboldKare/Scripts/PrecisionGrabber.cs

```diff
@@ -633,6 +638,11 @@ public bool TryUnfreeze() {
         }
         
         PhotonView otherView = hit.collider.GetComponentInParent<PhotonView>();
+        if(otherView.GetComponentInChildren<Freezable>()!=null)
+            {
+                otherView.RPC("UnfreezeRPC", RpcTarget.All); 

```

Comment:
This should be `otherView.RPC(nameof(Freezable.UnfreezeRPC), RpcTarget.All);` so the compiler can handle the string.